includes/js/checkout-block-frontend.js

```diff
@@ -0,0 +1,33 @@
+(()=>{
```

Comment:
@nidhitatosaniya You need to create a src folder with JS code and then create a build for it.